llvm/docs/ReleaseNotes.rst

```diff
@@ -108,6 +108,12 @@ Changes to the AArch64 Backend
   in ``standard`` being equal to ``bti+pac-ret+pc`` when ``+pauth-lr``
   is passed as part of ``-mcpu=`` options.
 
+* SVE and SVE2 have been moved to the default extensions list for ARMv9.0,
+  making them optional per the Arm ARM.  Existing v9.0+ CPUs in the backend that
+  support these extensions continue to have these features enabled by default
+  when specified via ``-mcpu=``.  The attribute ``"target-features"="+v9a"`` no
```

Comment:
nit: Did you mean `-march=` instead of `-mcpu=` ?

Reply:
I mean both. Adjusted to clarify this.